packages/maskbook/src/extension/options-page/DashboardComponents/TokenActionsBar.tsx

```diff
@@ -35,7 +36,7 @@ export interface TokenActionsMenuProps extends withClasses<never> {
     ) => void
 }
 
-export function TokenActionsMenu(props: TokenActionsMenuProps) {
+export const TokenActionsMenu = forwardRef((props: TokenActionsMenuProps) => {
```

Comment:
@Jack-Works 

Need your help, IDK why this can fix this warning:

react-dom.development.js?e52a:67 Warning: Function components cannot be given refs. Attempts to access this ref will fail. Did you mean to use React.forwardRef()?

Check the render method of `ForwardRef(Menu)`.

Reply:
https://github.com/DimensionDev/Maskbook/blob/bb5a2b580906f5505c6e142c05b43a77eea7dcfb/packages/maskbook/src/extension/options-page/DashboardComponents/TokenActionsBar.tsx#L49

```jsx
<div ref={ref}>
```

Reply:
does septs's solution works? @zhouhanseng

Reply:
no, so reverted. Now works fine.